src/components/Button.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+function Button( {type, clickHandler, children}) {
+    return (
+        <div>
+            <button
+            type={type}
+            onClick={clickHandler}
+            >{children}</button>
+        </div>
+    );
+}
```

Comment:
Dit is nu meer een doorgeefluik. Ten opzichte van het html button element zelf gebeurd er verder niets in dit component. Je had dus net zo goed direct kunnen gebruiken